src/app/public/modules/i18n/intl-number-formatter-options.ts

```diff
@@ -4,4 +4,5 @@ export interface SkyIntlNumberFormatterOptions {
   maximumFractionDigits?: number;
   currency?: string | null;
   currencyAsSymbol?: boolean;
+  currencySign?: string;
```

Comment:
Let's make this a string union type:
```
currencySign?: 'standard' | 'accounting';
```

Reply:
I thought about that.  I chose `string` to match the definition in the [Intl.NumberFormat](https://github.com/microsoft/TypeScript/blob/d699bcdaae9d7138eb29612d410e628154a254b6/src/lib/es5.d.ts#L4331) library, but I like being explicit about the values.  I guess the only downside would be if they add additional supported values to `Intl.NumberFormat`, we would need to add them to our definition as well.

If you think it is a good idea, I'm fine with changing it.

Reply:
Good points. Since this new property is exposed on "our" API, I'd rather lean on being more specific, instead of more generic. This would give us the flexibility to introduce our own "currency signs" that `Intl.NumberFormat` doesn't support, if needed.